src/fonduer/supervision/labeler.py

```diff
@@ -119,8 +187,15 @@ def drop_keys(self, keys):
             except AttributeError:
                 self.session.query(LabelKey).filter(LabelKey.name == key).delete()
 
-    def clear(self, train=False, split=0, **kwargs):
-        """Delete Labels of each class from the database."""
+    def clear(self, train, split, lfs):
+        """Delete Labels of each class from the database.
+
+        :param train: Whether or not to clear the LabelKeys.
```

Comment:
Consider renaming to `clear_labelkeys` or something related to the action inferred

Reply:
This needs to stay `clear`, as it is just implementing the `clear()` required by anything inheriting from `UDFRunner`